corehq/apps/sms/templates/sms/languages.html

```diff
@@ -9,7 +9,7 @@
 {% requirejs_main 'sms/js/sms_language_main' %}
 
 {% block stylesheets %}{{ block.super }}
-  <link rel="stylesheet" href="{% static 'jquery-ui/themes/redmond/jquery-ui.min.css' %}"/>
+  <link rel="stylesheet" href="{% static 'jquery-ui-built-themes/redmond/jquery-ui.min.css' %}"/>
```

Comment:
This can just be removed, since `SMSLanguagesView` properly uses the `use_jquery_ui` decorator.

Reply:
i think the css still needs to be included though?